ouroboros-consensus/src/Ouroboros/Consensus/MiniProtocol/ChainSync/Server.hs

```diff
@@ -156,4 +163,8 @@ chainSyncServerForReader tracer chainDB rdr =
 data TraceChainSyncServerEvent blk
   = TraceChainSyncServerRead        (Tip blk) (ChainUpdate blk (Point blk))
   | TraceChainSyncServerReadBlocked (Tip blk) (ChainUpdate blk (Point blk))
+  | TraceChainSyncRollForward
```

Comment:
Shouldn't we log a bit more, e.g. `Point blk`? Similar question for other messages.

Reply:
I added `Tip`.

Reply:
`Tip` will be confusing, on the server this will not be tip of the chain, I'd use just a `Point`.  @nfrisby what do you think?

Reply:
I was not able to find a way to get a `Point` for `TraceChainSyncRollForward`.  Can you suggest how I can get one?

Reply:
`Ouroboros.Network.Block.getTipPoint` should do it.